portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,17 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
+      <h1>Kwame Ampadu-Boateng's Portfolio</h1>
+      <p>Hello! My name is Kwame Ampadu-Boateng. I a currently a rising 
```

Comment:
Add a newline after `<p>`, makes it easier to see where the tag ends if the end tag is lined up with the start tag. e.g.

```
<p>
  some text.
  some more text
</p>
```